### PR TITLE
Permissions

### LINKED ISSUE #4: Update 'App permissions' page
Make sure the page matches the current set of permissions used by the app.

Maybe add a comment stating that only the contacts permission needs to be granted by the user, the rest is granted automatically.

### PR DESCRIPTION
**Pull request by artbristol**
Fixes #4

Easier to review if you look at the two commits separately as I reordered the permissions to match the Play store ordering.

I'm not convinced by my duplicate explanations for "External Storage Modification" vs "Storage".

### CODE REVIEW

File: docs/setup/permissions.md [hunk 1]
```diff
@@ -15,36 +18,55 @@ with careful consideration for both battery life and security - only opening con
...
 
-Attachments and settings can be loaded and saved onto external storage. In order read and write these files we need 
+### Read your contacts
+
+In order to mark e-mail that comes from your contacts and use your contacts to provide email address auto-completion, we
+need access to your contacts.
+
+### Modify your contacts
+
+In order to note that a contact has been emailed, we need write-access.
```
Comment (cketti) [inline docs/setup/permissions.md:28]: We no longer request the permission to write contacts. For the reason see https://developer.android.com/guide/topics/providers/contacts-provider#ObsoleteData

File: docs/setup/permissions.md [hunk 2]
```diff
@@ -15,36 +18,55 @@ with careful consideration for both battery life and security - only opening con
...
+
+## Storage
+
+### Read the contents of your USB storage
+### Modify or delete the contents of your USB storage
+
+Attachments and settings can be loaded and saved onto external storage. In order read and write these files we need
 access to do so.
```
Comment (cketti) [inline docs/setup/permissions.md:36]: We no longer request the storage permission (either for reading or writing). For all file system access outside of our app-private directory we use the storage access framework that lets users pick files/locations.

File: docs/setup/permissions.md [hunk 3]
```diff
@@ -15,36 +18,55 @@ with careful consideration for both battery life and security - only opening con
...
+## Photos / Media / Files
 
-#### Read Contacts
+### Read the contents of your USB storage
+### Modify or delete the contents of your USB storage
 
-In order to mark e-mail that comes from your contacts and use your contacts to provide email address auto-completion, we
-need access to your contacts
+Attachments and settings can be loaded and saved onto external storage. In order read and write these files we need
+access to do so.
```
Comment (cketti) [inline docs/setup/permissions.md:44]: Same as storage permission. Can be removed.

File: docs/setup/permissions.md [hunk 4]
```diff
@@ -15,36 +18,55 @@ with careful consideration for both battery life and security - only opening con
...
 
-#### Write Contacts
+## Other
 
-In order to note that a contact has been emailed we need write-access.
+### Read sync settings
 
-#### Control Vibration
+In order to fetch contacts properly we need this permission.
```
Comment (cketti) [inline docs/setup/permissions.md:50]: This permission is necessary to get the global auto-sync setting. See [ContentResolver#getMasterSyncAutomatically()](https://developer.android.com/reference/kotlin/android/content/ContentResolver#getMasterSyncAutomatically()).

File: docs/setup/permissions.md [hunk 5]
```diff
@@ -4,6 +4,9 @@ In order to perform various tasks and provide a cohesive email client, K-9 uses
 [Privacy Policy](https://k9mail.app/privacy) for information on how we handle personal information obtained via these 
 permissions.
 
+Only the Contacts permission will be requested by an Android system dialog when you use the app.
+The rest are granted automatically when you install K-9.
```
Comment (cketti) [inline docs/setup/permissions.md:8]: This is only true for Android 6 and above.

**Review by cketti — COMMENTED** [review]
The Play Store probably still shows a couple of permissions that we don't use anymore because K-9 Mail 5.600 is still available for Android 4.x devices (that can't use K-9 Mail 5.800).

File: docs/setup/permissions.md (hunk 4 above)
Comment (artbristol) [reply docs/setup/permissions.md]: I wasn't quite sure what to write here as I'm not 100% on which content providers the app uses

Comment (cketti) [reply docs/setup/permissions.md]: It has nothing to do with content providers. The global auto-sync setting (on my Android 11: *Settings app: Accounts > Automatically sync app data*) can be used as a flag to enable/disable background sync in K-9 Mail (see *Settings > General settings > Network > Background sync*).

**Pull request merged**